packages/@react-aria/slider/src/useMultiSlider.ts

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright 2020 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import {computeOffsetToValue} from './utils';
+import {HTMLAttributes, useRef} from 'react';
+import {mergeProps, useDrag1D} from '@react-aria/utils';
+import {MultiSliderProps} from '@react-types/slider';
+import {MultiSliderState} from '@react-stately/slider';
+import {useLabel} from '@react-aria/label';
+
+interface MultiSliderAria {
+  /** Props for the label element. */
+  labelProps: HTMLAttributes<HTMLElement>;
+
+  /** Props for the root element of the slider component; groups slider inputs. */
+  containerProps: HTMLAttributes<HTMLElement>;
+
+  /** Props for the track element. */
+  trackProps: HTMLAttributes<HTMLElement>;
+
+  /** Label ID for labling slider inputs. Should be passed on in `useSliderThumb`. */
+  labelId: string;
+}
+
+/**
+ * Provides behavior and accessibility for a slider component.
+ * 
+ * @param props Props for the slider.
+ * @param state State for the slider, as returned by `useMultiSliderState`.
+ * @param trackRef Ref for the "track" element.  The width of this element provides the "length"
+ * of the track -- the span of one dimensional space that the slider thumb can be.  It also
+ * accepts click and drag motions, so that the closest thumb will follow clicks and drags on
+ * the track.
+ */
+export function useMultiSlider(
```

Comment:
I'm thinking we should just call this `useSlider`. I don't think we need to reserve that name for anything else, so many that's simpler?